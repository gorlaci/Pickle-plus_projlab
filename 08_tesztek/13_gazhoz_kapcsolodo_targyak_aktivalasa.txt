create room r1
create student s
create mask m
create camambert c
create airfresher a
addperson s r1
activate c s
activate a s

status s
status m