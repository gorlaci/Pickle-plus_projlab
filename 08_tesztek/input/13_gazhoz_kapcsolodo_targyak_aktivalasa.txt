create room r1
create student s
create mask m
create camembert c
create airfresher a
addperson s r1
add c s
add a s
add m s
activate m s
activate c s
activate a s

status s
status m