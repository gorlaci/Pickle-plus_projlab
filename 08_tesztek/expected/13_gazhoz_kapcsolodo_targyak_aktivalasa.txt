
s
location: r1
itemsInHand: m, 
stunRemaining: 0

m
location: r1
holder: s
timeRemaining: 6
duration: 4
